code/modules/robotics/bot/secbot.dm

```diff
@@ -494,7 +504,7 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 				else if (target)		// make sure target exists
 					if (!IN_RANGE(src, src.target, 1))
 						src.moving = 0
-						navigate_to(src.target,(src.emagged >= 2) ? (ARREST_SPEED/2 * move_arrest_delay_mult) : (ARREST_SPEED * move_arrest_delay_mult), max_dist=50)
```

Comment:
Using the same speed regardless of emag status. Due to how much more relentless these guys are, it feels just fine.